[Hotfix] Fixing a level of zoom-related Blob issue

DeityLink opened the pull request:
So ghosts getting forced into blobs get their zoom level reset.

clusterfack commented:
This shouldn't be necessary, it should be done in mob/login()

clusterfack commented:
Also you fucked up in your blob hud update but @gbasood had merged it before I got to comment on it, thats why you encountered this issue in the first place

clusterfack commented:
It would be rude not to clarify so I'll comment now on that PR to show you how to fix it

clusterfack commented:
You also forget to have the blob hud reset on login

DeityLink commented:
* Added support for custom view sizes.
* Fixed blob hud no appearing on login

![viewsizes](https://cloud.githubusercontent.com/assets/7573912/15100042/713be022-1567-11e6-9997-75644411f0cf.png)

clusterfack commented on code/game/gamemodes/blob/blobs/core.dm:
```diff
@@ -141,6 +141,7 @@
 		C = new_overmind
 
 	if(C)
+		C.view = world.view
```
You forgot this line

DeityLink commented:
duh
